### PR TITLE
another troubleshooting instruction

### PR DESCRIPTION
**Pull request by smoothdeveloper**
and fix formatting

### CODE REVIEW

**Comment by KevinRansom** [comment]
Windows_NT Release_ci_part2 timed out --- @dotnet-bot test this please.

File: DEVGUIDE.md [hunk 1]
```diff
@@ -116,10 +116,10 @@ Note: if you face this error [#2351](https://github.com/Microsoft/visualfsharp/i
 
 >  error VSSDK1077: Unable to locate the extensions directory. "ExternalSettingsManager::GetScopePaths failed to initialize PkgDefManager for C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.exe".
 
-delete those folders:
+Or hard crash on launch ("Unkown Error"), delete those folders:
```
Comment (saul) [inline DEVGUIDE.md]: "Unknown error"

File: DEVGUIDE.md [hunk 2]
```diff
@@ -116,10 +116,10 @@ Note: if you face this error [#2351](https://github.com/Microsoft/visualfsharp/i
...
 
-* %appdata%\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
-* %appdata%\..\Local\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
+* `%appdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
+* `%appdata%\..\Local\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
```
Comment (saul) [inline DEVGUIDE.md]: %localappdata% instead?

File: DEVGUIDE.md [hunk 3]
```diff
@@ -116,10 +116,10 @@ Note: if you face this error [#2351](https://github.com/Microsoft/visualfsharp/i
 
 >  error VSSDK1077: Unable to locate the extensions directory. "ExternalSettingsManager::GetScopePaths failed to initialize PkgDefManager for C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.exe".
 
-delete those folders:
+Or hard crash on launch ("Unkown Error"), delete those folders:
 
-* %appdata%\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
-* %appdata%\..\Local\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
+* `%appdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
```
Comment (KevinRansom) [inline DEVGUIDE.md]: this guy is in Local too.

I'm okay with either %Appdata% or %LocalAppdata%..  TBH I didn't even know about %LocalAppdata%

**Comment by smoothdeveloper** [comment]
@saul @KevinRansom thanks for the reviews! it is now fixed.

File: DEVGUIDE.md [hunk 4]
```diff
@@ -116,10 +116,10 @@ Note: if you face this error [#2351](https://github.com/Microsoft/visualfsharp/i
 
 >  error VSSDK1077: Unable to locate the extensions directory. "ExternalSettingsManager::GetScopePaths failed to initialize PkgDefManager for C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\IDE\devenv.exe".
 
-delete those folders:
+Or hard crash on launch ("Unknown Error"), delete these folders:
 
-* %appdata%\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
-* %appdata%\..\Local\Microsoft\VisualStudio\15.0_<some number here>FSharpDev
+* `%appdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
+* `%localappdata%\..\Local\Microsoft\VisualStudio\15.0_(some number here)FSharpDev`
```
Comment (KevinRansom) [inline DEVGUIDE.md]: Sorry I wasn't very clear I think this should be: 
```
%localappdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev
%localappdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev
```
Folder looks like:
image

Comment (smoothdeveloper) [inline DEVGUIDE.md]: I see, the instructions worked in my case though, should I list all 4 folders?

Comment (KevinRansom) [inline DEVGUIDE.md]: No ... the listed folders should be the right ones.
This is actually incorrect:
%appdata%\Microsoft\VisualStudio\15.0_(some number here)FSharpDev

And this one has a redundant ..\Local  : 
%localappdata%\..\Local\Microsoft\VisualStudio\15.0_(some number here)FSharpDev

and I don't think the ticks add anything except making copy and paste trickier :-)

Comment (smoothdeveloper) [inline DEVGUIDE.md]: the ticks help in the display, not while editing.

ok, I hope this is good now :)

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

**Comment by KevinRansom** [comment]
@smoothdeveloper 

Thank you

Kevin

**Pull request merged**